docs/src/main/asciidoc/security-openid-connect-multitenancy.adoc

```diff
@@ -724,6 +724,18 @@ public class HelloResource {
 ----
 <1> The `io.quarkus.oidc.Tenant` annotation must be placed on either the resource class or resource method.
 
+[[TIP]]
+In the example above, authentication of the `sayHello` endpoint is enforced with the `@Authenticated` annotation.
+Alternatively, you can define the xref:security-authorize-web-endpoints-reference.adoc#authorization-using-configuration[HTTP Security policy] like in the example below:
```

Comment:
```suggestion
Alternatively, if you use an xref:security-authorize-web-endpoints-reference.adoc#authorization-using-configuration[HTTP Security policy] to secure the endpoint, then, for the `@Tenant` annotation be effective,  you must delay this policy's permission check as shown in the example below:
```

Reply:
I've tried to avoid using word `delay` as I think there can be negative feeling about _delaying security policies_ but it's true, that was it does and your suggestion makes things clearer, so I'll change it. Thank you